app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :user_id
+  has_many :likes, foreign_key: :user_id
```

Comment:
- [ ] Please replace `:user_id` with `:author_id` for `comments` and `likes` exactly as you did for posts check below :point_down:  my suggestion :slightly_smiling_face: 

```suggestion
  has_many :comments, foreign_key: :author_id
  has_many :likes, foreign_key: :author_id
```